geode-core/src/test/java/org/apache/geode/internal/util/ArgumentRedactorJUnitTest.java

```diff
@@ -21,20 +21,27 @@
 import static org.junit.Assert.assertTrue;
 
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 
+import org.apache.logging.log4j.Logger;
```

Comment:
Not used.

Reply:
Oops, meant to mark line 39

Reply:
Gah!  I really need to start committing via my IDE, so it can hit `Optimize Imports` for me.